app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  def latest_posts
+    Post.order(created_at: :desc).includes(:author).limit(2)
```

Comment:
- [ ] @OpondoG great work on your models kindly change the limit from 2 to 3 so that it returns the 3 most recent posts as per the requirement.
![image](https://user-images.githubusercontent.com/61408860/216679417-fd98950c-e14a-4713-83b1-a8b7cb5df2ea.png)